lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mod.rs

```diff
@@ -273,6 +283,8 @@ impl InvertedIndex for MmapInvertedIndex {
     }
 
     fn get_token_id(&self, token: &str, hw_counter: &HardwareCounterCell) -> Option<TokenId> {
+        let hw_counter = self.make_conditioned_counter(hw_counter);
```

Comment:
too much overhead for such a simple function, isn't it?

Reply:
I woulf rather have `if self.is_on_disk` for such cases

Reply:
Yeah good point. Fixed!